tests/e2e/package.json

```diff
@@ -6,11 +6,11 @@
     "lint": "eslint src/*.js ./*.js"
   },
   "dependencies": {
-    "@chainsafe/dappeteer": "^2.4.0",
+    "@chainsafe/dappeteer": "^3.0.0",
```

Comment:
Compatibility of this version may be have bugs, specific details( https://github.com/ChainSafe/dappeteer/issues/155 ). I am considering other ways as backup plan.